System.Runtime.UnitTests/CallerArgumentExpressionAttributeTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Runtime.CompilerServices;
+using nanoFramework.TestFramework;
+
+namespace System.Runtime.UnitTests
+{
+    [TestClass]
+    public class CallerArgumentExpressionAttributeTests
+    {
+        public string Field = "Field value";
+
+        public string Property => "Property value";
+
+        [TestMethod]
+        public void CallerArgumentExpression_sets_ParameterName_from_field()
```

Comment:
Project code style calls for camel case method name. I know most of the mscorlib unit tests are using this naming style, but we should default to the appropriate one when adding new tests. Moreover, this is a brand-new project, so let's have it by the book, OK? 😉